build-test.xml

```diff
@@ -2725,7 +2725,6 @@ httpLoggingEnabled=&quot;false&quot;</echo>
 	<macrodef name="prepare-captcha-configuration">
```

Comment:
@jeffwu0724 @Debburiti This prepare-captcha-configuration marcodef should be removed.